test_query_vaults_async is flaky

## Problem Definition
Test `test_query_vaults_async` is flaky

```=================================== FAILURES ===================================
______________ test_query_vaults_async[True-ethereum_modules0-1] _______________

rotkehlchen_api_server = <rotkehlchen.api.server.APIServer object at 0x110906e50>
ethereum_accounts = ['0xdB90d9010321694D34D1f568d1d225EF769DD48E']

...

_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

>   ???
E   gevent.timeout.Timeout: 10 seconds
```

See here: https://github.com/rotki/rotki/pull/1027/checks?check_run_id=709592741

--- a/rotkehlchen/tests/api/test_makerdao_vaults.py
+++ b/rotkehlchen/tests/api/test_makerdao_vaults.py
@@ -140,7 +140,7 @@ def test_query_vaults_async(rotkehlchen_api_server, ethereum_accounts):
         "makerdaovaultsresource",
     ), json={'async_query': True})
     task_id = assert_ok_async_response(response)
-    outcome = wait_for_async_task(rotkehlchen_api_server, task_id)
+    outcome = wait_for_async_task(rotkehlchen_api_server, task_id, timeout=30)
     assert outcome['message'] == ''
     vaults = outcome['result']
     _check_vaults_values(vaults, ethereum_accounts[0])
@@ -150,7 +150,7 @@ def test_query_vaults_async(rotkehlchen_api_server, ethereum_accounts):
         "makerdaovaultdetailsresource",
     ), json={'async_query': True})
     task_id = assert_ok_async_response(response)
-    outcome = wait_for_async_task(rotkehlchen_api_server, task_id)
+    outcome = wait_for_async_task(rotkehlchen_api_server, task_id, timeout=30)
     assert outcome['message'] == ''
     details = outcome['result']
     _check_vault_details_values(details)

